src/metadataGeneration/tsoa.ts

```diff
@@ -182,6 +184,10 @@ export namespace Tsoa {
     dataType: 'buffer';
   }
 
+  export interface StreamType extends TypeBase {
```

Comment:
You can avoid adding a new type (and therefore, a lot of code) if you reuse `BufferType`.

Reply:
Good idea. I've pushed a new commit to reuse `BufferType`.